bin/pygrb/pycbc_pygrb_page_tables

```diff
@@ -113,34 +113,53 @@ def load_missed_found_injections(hdf_file, ifos, bank_file, snr_threshold=None,
     for param in ['mass1', 'mass2', 'spin1z', 'spin2z']:
         found_data['rec_'+param] = \
             np.array(bank_file[param])[inj_data['network/template_id']]
-    found_data['time_diff'] = \
-        found_data['tc'] - inj_data['network/end_time_gc'][...]
+    if 'network/end_time_gc' in inj_data:
+        found_data['time_diff'] = \
+                found_data['tc'] - inj_data['network/end_time_gc'][...]
+    else:
+        found_data['time_diff'] = np.array([])
     found_data['rec_mchirp'] = mchirp_from_mass1_mass2(
         found_data['rec_mass1'],
         found_data['rec_mass2'])
     # Recovered RA and Dec
-    found_data['rec_ra'] = inj_data['network/ra'][...]
-    found_data['rec_dec'] = inj_data['network/dec'][...]
+    if 'network/ra' and 'network/dec' in inj_data:
+        found_data['rec_ra'] = inj_data['network/ra'][...]
+        found_data['rec_dec'] = inj_data['network/dec'][...]
+    else:
+        found_data['rec_ra'] = np.array([])
+        found_data['rec_dec'] = np.array([])
```

Comment:
Please undo this change.  If there are no found injections the `return` above prevents the code from getting here.